lib/feature/home/create_task/bloc/create_task_state.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter/foundation.dart';
+
+@immutable
+abstract class CreateTaskState extends Equatable {
+  @override
+  List<Object> get props => [];
+
+  @override
+  String toString() {
+    return this.runtimeType.toString();
+  }
+}
+
+/// The initial state for task creation
+class CreateTaskInitial extends CreateTaskState {}
```

Comment:
Maybe we should prefer more descriptive names for the initial state instead of initial. AwaitUserInput might be a better name because it describes the state and you may need to hold the already entered fields here to preserve the state in case the widget is rebuilt (you already use TextEditingController, but if you have a lifted state and now have 2 pages/screens you'll want to pass information while still in this awaiting user input state).

Reply:
Thanks :) I renamed the state